src/Main.java

```diff
@@ -1,5 +1,117 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("задача 1");
+        int depositAmount = 0;
+        int total = 0;
+        while (total <= 2459000) {
+            total = total + 15000;
+            depositAmount++;
+            System.out.println("Месяц " + depositAmount + ", сумма накоплений  " + total + " рблей");
+        }
+        System.out.println("Задача 2");
+        for (int i = 0; i <= 10; i++)
```

Comment:
```suggestion
        for (int i = 1; i <= 10; i++)
```
по заданию от 1, а не от 0